gammapy/image/models/tests/test_models.py

```diff
@@ -19,7 +19,7 @@ def test_sky_point_source():
         lat_0='45 deg',
     )
     lon = [1, 1.1] * u.deg
-    lat = 45 * u.deg
+    lat = [45, 45.2] * u.deg
     val = model(lon, lat)
     assert val.unit == 'sr-1'
```

Comment:
Could you add the test from #1399 again? So evaluate the `SkyPointSourceModel` on a grid and check that the result integrates to unity?